src/routes/(console)/project-[region]-[project]/databases/database-[database]/collection-[collection]/document-[document]/settings/+page.svelte

```diff
@@ -29,7 +29,7 @@
                     $doc.$databaseId,
                     $doc.$collectionId,
                     $doc.$id,
-                    $doc.data,
+                    $doc,
```

Comment:
Maybe this should be undefined so that only permissions are updated. That's what the behavior before essentially was.

Reply:
Done.